library/std/src/thread/mod.rs

```diff
@@ -738,6 +738,8 @@ pub fn panicking() -> bool {
 ///
 /// This function is blocking, and should not be used in `async` functions.
 ///
+/// The behavior when given a zero duration is platform-specific.
```

Comment:
What is someone reading this documentation supposed to do with this fact?

Reply:
Would it help to clarify the actual differences between platforms, see my more complete comment [above](https://github.com/rust-lang/rust/pull/79849#discussion_r539569278)

Reply:
Perhaps we could direct them to `yield_now` instead?